autoprotocol/types/protocol.py

```diff
@@ -17,17 +17,19 @@
 AMOUNT_CONCENTRATION = Union[str, Unit]
 CONCENTRATION = Union[str, Unit]
 FLOW_RATE = Union[str, Unit]
-FREQUENCY = Union[str, Unit]
-LENGTH = Union[str, Unit]
+FREQUENCY = Union[str, Unit]  # hertz, rpm
+LENGTH = Union[str, Unit]  # mm
```

Comment:
nit: make similar to `WAVELENGTH` comment, with expanded unit notation

```suggestion
LENGTH = Union[str, Unit]  # millimeter
```